src/blog/_posts/2017-11-08-opera-49.md

```diff
@@ -0,0 +1,98 @@
+---
+title: What’s new in Chromium 62 and Opera 49
+authors:
+- daniel-bratell
+intro: 'Opera 49 (based on Chromium 62) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To find out what’s new for users, see our [Desktop blog post](http://blogs.opera.com/desktop/). Here’s what it means for web developers.'
+tags:
+- html
+- css
+- dom
+- javascript
+- opera
+license: cc-by-3.0
+---
+
+Opera 49 (based on Chromium 62) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To
+find out what’s new for users, see our
+[Desktop blog post](http://blogs.opera.com/desktop/).
+Here’s what it means for web developers.
+
+
+## Network Quality Estimator API
+
+There are times when a web page want to adapt to the user's network. To
+help a web page with that information there is now
+[Network Infomation API](https://wicg.github.io/netinfo/) which can give a
+rough indication of the user's network, including bandwidth and round-trip time.
+
+This is still under development but the parts that exist today, `rtt`,
+`effectiveType` and `downlink`, should be enough for most use cases.
+
+The plan is that these signals will also become available as HTTP request
+headers and enabled via Client Hints.
+
+## OpenType Variable Fonts
+
+Previously a font file could only contain one weight/variant of a font. So you
+had one file for bold, one file for semi-bold, one for stretched, one
+for normal and so on. With
+[OpenType Font Variations](https://medium.com/@tiro/https-medium-com-tiro-introducing-opentype-variable-fonts-12ba6cd2369)
+you can combine those
+variants efficiently in a single file. This should allow web pages to be smaller
+and load faster than before, while also giving web pages access to an
+infinite number of font variants (not that I recommend using more than a [few
```

Comment:
Maybe not use "I" here, but maybe something like "using more than a few in page cannot be recommend though"?

Reply:
I want to keep this a bit personal since it's an opinion.